### ISSUE TITLE
Swift deprecations

### ISSUE THREAD

**Issue by JWiseCoder** [vcooley/nativescript-input-mask#4]
The swift code in the input-mask-ios repository the podfile in this links to uses deprecated Swift code, mostly involving string manipulation (e.g. String.characters)

It will compile and run (albeit with a lot of warnings), but can this be fixed so it will also run in the future?

**Comment by vcooley** [comment]
Yes this is on my todo list. Some of the bindings have changed in the latest version of the native dependency, so it will take some refactoring.
